ResearchKit/Common/ORKHelpers.h

```diff
@@ -180,6 +180,17 @@ ORKEqualFileURLs(NSURL *url1, NSURL *url2) {
     return ORKEqualObjects(url1, url2) || ([url1 isFileURL] && [url2 isFileURL] && [[url1 absoluteString] isEqualToString:[url2 absoluteString]]);
 }
 
+ORK_INLINE BOOL
+ORKCurrentLocalePresentsFamilyNameFirst() {
+    NSString * language = [[[NSLocale preferredLanguages] firstObject] substringToIndex:2];
+    static dispatch_once_t onceToken;
+    static NSArray *familyNameFirstLangs = nil;
+    dispatch_once(&onceToken, ^{
+        familyNameFirstLangs = @[@"zh",@"ko",@"ja"];
+    });
+    return (language != nil) && [familyNameFirstLangs containsObject:language];
+}
+
```

Comment:
As far as I understand, `inline` keyword would actually copy the code to the places where it get referenced. It causes `static dispatch_once_t onceToken` and `static NSArray *familyNameFirstLangs` to be duplicated. Since we want `familyNameFirstLangs` to be only constructed once, do not use `inline` with this function. Do something similar to `NSDateComponents *ORKTimeOfDayComponentsFromDate(NSDate *date);`.